src/views/Home.vue

```diff
@@ -1,101 +1,99 @@
+              Cozy at Home - via Ambient Intelligence
+            </v-list-item-subtitle>
 
-          <v-card-text id="welcome-text">
-            Review your home timeline for notable moments.
-            Configure input sensors and camers for Ambianic to observe.
-            Share, purge or backup your data
-            - it never slips out of your control.
-          </v-card-text>
+            <v-card-text id="welcome-text">
+              <p v-if="!hasSetupSystem">
+                Let's setup your system
+              </p>
+              <p v-else>
+                Control your Ambianic Edge Appliances from your console.
+              </p>
+            </v-card-text>
 
-          <v-card-actions>
-            <v-btn
-              rounded
-              color="pink darken-4"
-              dark
-              data-cy="timeline"
-              class="ma-2 white--text"
-              :to="'timeline'"
-              id="btn-timeline"
-            >
-              View Timeline
-              <v-icon
-                right
+            <div v-if="hasSetupSystem">
+              <v-btn
+                rounded
+                color="pink darken-4"
+                dark
+                data-cy="timeline"
+                class="ma-2 white--text"
+                :to="'timeline'"
+                id="btn-timeline"
               >
-                mdi-history
-              </v-icon>
-            </v-btn>
-            <v-spacer />
-            <v-btn
-              id="btn-settings"
-              rounded
-              color="pink darken-4"
-              dark
-              class="ma-2 white--text"
-              :to="'settings'"
-              data-cy="settings"
-            >
-              Settings
-              <v-icon
-                right
+                Open Edge Console
+              </v-btn>
+            </div>
+            <div v-else>
+              <v-btn
+                rounded
+                color="pink darken-4"
+                dark
+                data-cy="timeline"
+                class="ma-2 white--text"
+                :to="'onboarding'"
+                id="btn-timeline"
               >
-                mdi-settings
-              </v-icon>
-            </v-btn>
-          </v-card-actions>
-        </v-card>
-      </v-row>
-      <v-row
-        align="end"
-        justify="center"
-        no-gutters=""
-      >
-        <v-col>
-          <update-notification class="mx-auto" />
-        </v-col>
-      </v-row>
-    </v-container>
-  </v-content>
+                Continue Setup
+                <v-icon right>
+                  mdi-arrow-right
+                </v-icon>
+              </v-btn>
+            </div>
+          </v-list-item-content>
+        </v-list-item>
+      </v-card>
+    </div>
+  </div>
 </template>
 
 <script>
-import UpdateNotification from '@/components/UpdateNotification'
-
 export default {
   name: 'Home',
-  components: {
-    UpdateNotification
```

Comment:
@vickywane why did you get rid of the updatenotification? It is what allows us to push new PWA versions to production with the correct UX.

Reply:
I thought it wasn't being used. I would restore it back

Reply:
It is used all the time. The user gets a friendly pop-up that their app has a new version. They can click refresh when they are ready to load it. It's been working nicely for quite some time. 

Which reminds me that we need to cover it with tests so developers get a flag if they remove it again or mess it up. I'll open a separate issue for that. Not part of this PR.